core/datastore/src/main/java/at/marki/moviedb/core/datastore/PreferenceItems.kt

```diff
@@ -22,6 +22,8 @@ object PreferenceItems {
     // Favorites Keys
 
     val favoriteIdsItem by lazy {
+        //REVIEW: instead of parsing the list of ids to json you could have
+        // used a stringSetPreferencesKey
```

Comment:
I still have to improve my "Datastore Wrapper" (it isn't really a wrapper.. yet)